src/App/Fossa/FossaAPIV1.hs

```diff
@@ -325,7 +317,7 @@ instance FromJSON Issue where
     Issue <$> obj .: "id"
            <*> obj .:? "priorityString"
            <*> obj .: "resolved"
-           <*> obj .: "revisionId"
+           <*> obj .:? "revisionId" .!= "unknown project"
```

Comment:
[Question]: What's the background for this change?

Reply:
Added a comment there; essentially, the shape of the response annoyingly changes when we ask for issues for a VPS project